g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/validation/ParameterValidation.java

```diff
@@ -34,6 +33,7 @@ public void validate() throws ValidationException {
 		validateComponent(request);
 		validateKey(request);
 		validateLocale(request);
+		validateLanguage(request);
```

Comment:
Is this duplicate validation because there has already called validate in TranslationWithPatternAction.java ?

Reply:
it only can validate the path variable or query parameter, can't validate the post content